hakyll-shortcut-links.cabal

```diff
@@ -31,6 +32,8 @@ common common-options
                        -fhide-source-paths
                        -Wmissing-export-lists
                        -Wpartial-fields
+  if impl(ghc >= 8.8.1)
+    ghc-options:       -Wmissing-deriving-strategies
```

Comment:
Let's error on missing deriving strategies, not just warn:

https://github.com/kowainik/tomland/blob/d9b7a1dc344e41466788fe00d0ea016f04629ade/tomland.cabal#L58-L60